numpy/core/src/umath/loops.c.src

```diff
@@ -1014,18 +1015,57 @@ NPY_NO_EXPORT NPY_GCC_OPT_3 void
     UNARY_LOOP_FAST(@type@, @type@, *out = in > 0 ? 1 : 0);
 }
 
+/* Libdivide only supports 32 and 64 bit types
+ * We try to pick the best possible one */
+#if NPY_BITSOF_@TYPE2@ <= 32
+#define libdivide_@type@_t libdivide_u32_t
+#define libdivide_@type@_gen libdivide_u32_gen
+#define libdivide_@type@_do libdivide_u32_do
+#else
+#define libdivide_@type@_t libdivide_u64_t
+#define libdivide_@type@_gen libdivide_u64_gen
+#define libdivide_@type@_do libdivide_u64_do
+#endif
```

Comment:
Just realized, if there is ever hardware with long long > 64bit we are going to have a problem.  If there is hardware like that, we have to fix this in the 1.20 release. If we are not worried about such existence, we still should probably anticipate it, or at least put a `#error "Uhoh!"` in here.

Reply:
Sounds good, thanks. I'll see if I can test it.